src/app/core/services/basic-rundown-state.service.ts

```diff
@@ -17,12 +18,16 @@ export class BasicRundownStateService implements OnDestroy {
   private readonly isLoadingSubject: BehaviorSubject<boolean>
   private isLoading = true
 
+  private readonly logger: Logger
+
   constructor(
     private readonly basicRundownService: BasicRundownService,
     private readonly rundownEventObserver: RundownEventObserver,
     private readonly connectionStatusObserver: ConnectionStatusObserver,
-    private readonly basicRundownEntityService: BasicRundownEntityService
+    private readonly basicRundownEntityService: BasicRundownEntityService,
+    logger: Logger
   ) {
+    this.logger = logger.tag('BasicRundownStateService')
```

Comment:
Are we interested in the syntax: `logger.tag(BasicRundownStateService.name)`?

Reply:
If we want to be able to have these readable in production, then no.